frontend/src/metabase/visualizations/shared/settings/cartesian-chart.ts

```diff
@@ -195,7 +195,7 @@ export const getSeriesOrderVisibilitySettings = (
   }));
 };
 
-export const getDefaultYAxisTitle = (metricNames: string[]) => {
+export const getDefaultYAxisTitle = (metricNames: (string | undefined)[]) => {
```

Comment:
It's quite confusing when `metrics` is an array of `string | null` but their names are an array of `string | undefined`. To be clear, I don't have any suggestion here, but I just want to voice that this might need a cleanup, but not now.